lib/models.ts

```diff
@@ -13,7 +13,13 @@ export interface CDE {
     InterLexID: string;
 }
 
-export interface Repository {
+export interface CustomDictionaryField {
+    VariableName: string;
+    Title: string;
```

Comment:
From the figma mockups Title and Variable Name seem to be mandatory. I think it would make more sense if Title would match with the equivalent attribute in the CDE (PreciseCDEAbbrev)

Reply:
@afonsobspinto I'm not sure we should take Figma as a starting point of our models. The CDE dialog can have it's own internal (technical) model that maps to the figma design